scripts/update-version.sh

```diff
@@ -46,3 +46,9 @@ fi
 step "Updating copyright year to ${YEAR}…"
 
 sed -i '' -E "s/© ([0-9]{4})[–-][0-9]{4}/© \\1–${YEAR}/g" LICENSE.md docs/jazzy.yml Sources/MapboxDirections/Info.plist
+
+# Starting with first release candidate, we should not have breaking changes.
+if [[ $1 == *rc.1 ]]; then
```

Comment:
I would suggest updating the baseline for GA releases too. Not sure about patch releases (this suggestion includes them, maybe it shouldn't and we need `&& $SHORT_VERSION == *.0`). It isn't expected that we add new APIs in RCs, but it might happen.
```suggestion
if [[ $1 == *rc.1 || $SHORT_VERSION == $SEM_VERSION ]]; then
```